### ISSUE TITLE
Internal URL Served to Media Browser

### ISSUE THREAD

**Issue by Robert-MacWha** [music-assistant/client#19]
`MusicAssistantClient.get_media_item_image_url` is serving an internal URL to the "https://github.com/music-assistant/hass-music-assistant" media browser, causing local artwork for artists, albums, tracks, and (I assume) playlists to fail to load.  This seems to be a problem with `get_image_url`, using Music Assistant's internal IP for requests made from a browser.

Could we change `get_image_url` to use the `hassio_ingress` API?  I'm not sure where else the method might be used, but this seems like it would fix the issue at hand.

Original Issue: https://github.com/music-assistant/hass-music-assistant/issues/2045

**Comment by marcelveldt** [comment]
Yes there is a feature request for that but is not that simple as all the required plumbing in HA does not exist fo rthe media browser, only for single images on a media player.

BTW: Please use the MA support channels next time to get support so we do not have to track 10000 places. Thanks!
